app/api/hateoas/HateoasLinks.scala

```diff
@@ -16,34 +16,34 @@
 
 package api.hateoas
 
+import api.hateoas
 import api.models.domain.{BusinessId, Nino, TaxYear}
-import api.models.hateoas.Link
-import api.models.hateoas.Method.{DELETE, GET, POST, PUT}
-import api.models.hateoas.RelType._
+import Method.{DELETE, GET, POST, PUT}
+import RelType._
 import config.AppConfig
 
 trait HateoasLinks {
 
   def retrieveAnnualSubmission(appConfig: AppConfig, nino: Nino, businessId: BusinessId, taxYear: TaxYear): Link =
-    Link(href = annualSubmissionUri(appConfig, nino, businessId, taxYear), method = GET, rel = SELF)
+    hateoas.Link(href = annualSubmissionUri(appConfig, nino, businessId, taxYear), method = GET, rel = SELF)
```

Comment:
```suggestion
    Link(href = annualSubmissionUri(appConfig, nino, businessId, taxYear), method = GET, rel = SELF)
```
Can we do this for all the hateoas.Link?